app/components/post_body_additional_content/post_body_additional_content.js

```diff
@@ -124,11 +125,16 @@ export default class PostBodyAdditionalContent extends PureComponent {
                 imageUrl = `https://i.ytimg.com/vi/${videoId}/hqdefault.jpg`;
                 ImageCacheManager.cache(null, `https://i.ytimg.com/vi/${videoId}/default.jpg`, () => true);
             } else {
-                const {data} = await this.props.actions.getRedirectLocation(link);
-
                 let shortenedLink;
-                if (data && data.location) {
-                    shortenedLink = data.location;
+
+                if (expandedLink) {
+                    shortenedLink = expandedLink;
```

Comment:
Minor nit, perhaps we could stick with a consistent name here instead of mixing `shortenedLink` and `expandedLink` since those mean the opposite things to me.